src/server/models/cacheable_queries/organization.js

```diff
@@ -18,23 +19,16 @@ const organizationCache = {
     }
     return getConfig("TWILIO_MESSAGE_SERVICE_SID", organization);
   },
-  getMessageServiceAuth: async (organization, contact, messageText) => {
-    // Note organization won't always be available, so we'll need to conditionally look it up based on contact
-    if (messageText && /twilioapitest/.test(messageText)) {
-      return {
-        messagingServiceSid: "fakeSid_MK123",
-        authToken: "foobar",
-        apiKey: "foobarbaz"
-      };
-    }
-    return {
-      messagingServiceSid: getConfig(
-        "TWILIO_MESSAGE_SERVICE_SID",
-        organization
-      ),
-      authToken: getConfig("TWILIO_API_KEY", organization),
-      apiKey: getConfig("TWILIO_AUTH_TOKEN", organization)
-    };
+  getTwilioAuth: async organization => {
+    const multiOrg = getConfig("TWILIO_MULTI_ORG");
+    const hasOrgToken = hasConfig("TWILIO_AUTH_TOKEN_ENCRYPTED", organization);
+    const authToken = multiOrg && hasOrgToken
```

Comment:
That's actually how I had it originally, but then I changed it, with the thinking that if someone disables TWILIO_MULTI_ORG, the intuitive behavior would be to revert to always using thing global credentials.

That's probably an edge case though and I'm fine with changing it to what you suggest.

Reply:
Actually, one question for you. Should it be:
`const authToken = hasOrgToken 
? symmetricDecrypt(getConfig("TWILIO_AUTH_TOKEN_ENCRYPTED", organization))
: getConfig("TWILIO_AUTH_TOKEN", organization)`

Or:
`const authToken = hasOrgToken 
? symmetricDecrypt(getConfig("TWILIO_AUTH_TOKEN_ENCRYPTED", organization))
: getConfig("TWILIO_AUTH_TOKEN")`

The former would allow folks to store unencrypted auth tokens in the database, which I think is bad practice, but perhaps they don't care (and they would have to manually insert that value in the database).

The latter would always fallback to the environment variable if the encrypted token wasn't present.

Reply:
I suppose I'm ok with either.  I think if we do the latter (forbidding non-encrypted tokens in the db), then we should really write up some good documentation on how to use your dev-tools/ commands and add something to the database.